### ISSUE TITLE
How to add Day, Week, Month view?

### ISSUE THREAD

**Issue by smitaAcellere** [patchthecode/JTAppleCalendar#229]
IMPORTANT - BEFORE CREATING YOUR ISSUE PLEASE READ THE FOLLOWING:

1. Are you using the latest version of JTAppleCalendar? Latest version is currently 6.0.5
2. Is your question already answered in the tutorials and guides —> https://patchthecode.github.io/
3. Looking for some feature you cant find? Than have you already looked at the *UserInteractionFunctions.swift* file?
It has every function that a user is able to do.

Like this project? Consider supporting by leaving a star rating. :)

Thanks,
Jay

**Comment by patchthecode** [comment]
What is a:
1. Day view
2. Month view
3. Year view
?

**Comment by smitaAcellere** [comment]
you can refer attached files.. I wanted to do design like attached files..

![trnear_cal_daily_master_comp_test](https://cloud.githubusercontent.com/assets/17287041/21170479/6b6d9204-c1eb-11e6-86af-fbaa3a4ee297.gif)
![trnear_cal_master_comp_test](https://cloud.githubusercontent.com/assets/17287041/21170481/6bbd6176-c1eb-11e6-9f2c-47e0cebe76c1.gif)

**Comment by patchthecode** [comment]
hmm. This library can build any calendar so long it is a traditional grid style.

Well some comments on the views you have provided.

1. The Day view looks like a regular UITableView. The library will not be able to build something like that. But I think this can be built with a UITableView.
2. The Week view looks like it has times attached to it. It looks like what I call a "Scheduler". It will not be able to build this. The only part it is able to build is this: 
<img width="457" alt="screen shot 2016-12-13 at 9 33 11 pm" src="https://cloud.githubusercontent.com/assets/2439146/21170705/cdef2282-c17b-11e6-8964-9de2bb276250.png">
3. The year view is being worked on. 

In short, i do not think this library can meet your needs now, unfortunately. :(

**Comment by smitaAcellere** [comment]
Thanks yaaaa. It means a lot. 
I will work on it in another way. :)

**Comment by patchthecode** [comment]
cool cheers 🍺
